DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -86,6 +86,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', $config['tags']['enabled']);
         if ($config['tags']['enabled']) {
+            // only load tag_invalidate if tags are enabled
+            $loader->load('tag_invalidate.xml');
```

Comment:
i wonder if we even need to have two separate configuration files. the tag_listener.xml file already contains other services.

@ddeboer what do you think about renaming the file to `cache_tags.xml` or something and have all configuration in there?